pages/blog/posts/post5.md

```diff
@@ -0,0 +1,163 @@
+---
+date: 2024-09-25
+title: Subcontracting Workflow in ERPNext
+category: Article
+author: Ninad Parikh
+description: Understanding the complete Subcontracting Workflow and different Scenarios for Subcontracting.
+tags:
+  - Subcontracting
+  - ITC-04
+
+og_title: Blog | Subcontracting Workflow in ERPNext
+og_url: https://docs.indiacompliance.app/blog/posts/post5
+---
+
+<PostDetail>
+
+The complete guideline of how subcontracting works in ERPNext is provided
+[here](https://docs.erpnext.com/docs/user/manual/en/subcontracting).
+
+In this blog we will explore the various documents involved in the
+subcontracting process and delve into different scenarios, offering a clear
+ understanding of its practical applications and workflows.
+
+## What is Subcontracting ??
+
+Subcontracting, or job work, allows businesses to send raw materials or semi-
+finished goods to external vendors for further processing or assembly, enabling
+efficient resource utilization. This approach helps companies focus on core
+ activities while leveraging external expertise to reduce production costs and
+  increase flexibility in scaling operations.
+
+## Subcontracting Workflow
+
+### Items
```

Comment:
Set the base for what is ITC04, when it is required.
What are the tables, which ones are we using.

Refer to table numbers in transactions explained below.